spec/features/create_work_spec.rb

```diff
@@ -62,7 +62,7 @@
       find('body').click
       choose('work_visibility_open')
       expect(page).to have_content('Please note, making something visible to the world (i.e. marking this as Public) may be viewed as publishing which could impact your ability to')
-      check('agreement')
+
 
```

Comment:
This seems odd. Why don't we have to agree to the TOS here?

Reply:
Just removed agreement checkout (within hyrax config) for a faster asset upload for now.